hadoop-cloud/pom.xml

```diff
@@ -171,6 +171,15 @@
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-cos</artifactId>
         </exclusion>
+        <!--
+          SPARK-48867: `hadoop-huaweicloud` is rarely used, the users have to add it separately.
```

Comment:
Does this PR introduce any user-facing change?
No.

isn't this a user-facing change for those that are using huaweicloud?

Reply:
@bjornjorgensen,

Good catch! I have forgotten about this part. Done.